### PR TITLE
docs(grid): new code tab

### LINKED ISSUE #3508: Grid docs: Add new "Code" page to grid docs
- Add new "Code" tab to 2x Grid section, follow structure from other guideline & component pages
- Remove get started section from code tabs on the following Guidelines pages
  - color
  - Motion
  - Themes
  - Typography

### PR DESCRIPTION
**Pull request by alisonjoseph**
Closes #3508

- New Code tab for 2x Grid
- Remove get started section from code tabs on color, motion, themes & typography & anchor links

https://carbondesignsystem-git-fork-alisonj-668941-carbon-design-system.vercel.app/guidelines/2x-grid/code

~Question around anchor links on the color, motion, themes & typography pages now that there are only 2, do they stay or get removed?~

### CODE REVIEW

**Comment by kingtraceyj** [comment]
@alisonjoseph i think you're right—if there are only 2 anchor links, we can take them out. if there are 3 or more we should leave them in. so we can remove the color, motion, themes & typography anchor links on the code tabs.

**Comment by alisonjoseph** [comment]
updated @kingtraceyj

**Comment by kingtraceyj** [comment]
@alisonjoseph is the [the helpers on storybook](https://react.carbondesignsystem.com/?path=/story/helpers-hideatbreakpoint--hide-at-breakpoint&_ga=2.194842606.270246924.1682613159-223315952.1682613159) broken in both the API & helpers section and the resource card?

File: src/pages/guidelines/2x-grid/code.mdx [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+title: 2x Grid
+description:
+  The grid is the framework for all the visual elements and typography at IBM.
+  It’s fundamental to everything we design.
+tabs: ['Overview', 'Implementation', 'Code']
+---
+
+<PageDescription>
+
+The Carbon Grid pacakge helps developers build digital experiences with the 2x
```
Comment (sstrubberg) [inline src/pages/guidelines/2x-grid/code.mdx:11]:
```suggestion
The Carbon Grid package helps developers build digital experiences with the 2x
```

**Comment by sstrubberg** [comment]
Reminder from our conversation we just had to add more progressive disclosure to the ResourceCards and to drop the Flexgrid example to a legacy section or something like that.

**Review by tay1orjones — COMMENTED** [review]
> Remove get started section from code tabs on color, motion, themes & typography & anchor links

Should we keep [spacing](https://carbondesignsystem-git-fork-alisonj-668941-carbon-design-system.vercel.app/guidelines/spacing/code) that references the `@carbon/layout` package installation?

**Comment by alisonjoseph** [comment]
@tay1orjones oops, not sure how I missed that one, I'll remove it from spacing too.

**Review by andreancardona — APPROVED** [review]
looks good!

**Review by sstrubberg — APPROVED** [review]

**Comment by alisonjoseph** [comment]
@kingtraceyj I just updated it to group it with the cards above so its not floating on its own, I don't think it needs its own section, but happy to update if you feel strongly about it.

**Review by tay1orjones — APPROVED** [review]

**Review by kingtraceyj — APPROVED** [review]
looks good! 💥

**Pull request merged**